build.gradle

```diff
@@ -117,6 +117,7 @@ allprojects {
     configurations.all {
         resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib:1.9.10"
         resolutionStrategy.force "org.jetbrains.kotlin:kotlin-stdlib-common:1.9.10"
+        resolutionStrategy.force "net.bytebuddy:byte-buddy:1.14.9"
```

Comment:
OS core bumps Jackson version which in turn bumps byte-buddy version to 1.14.9. But it seems OS test framework is still using old version. Will remove this change if the conflict is gone.

```
net.bytebuddy:byte-buddy:1.14.7 -> 1.14.9
\--- org.opensearch.test:framework:3.0.0-SNAPSHOT:20240315.134538-2101
     \--- testCompileClasspath
```